types/classnames/index.d.ts

```diff
@@ -6,18 +6,11 @@
 //                 Sean Kelley <https://github.com/seansfkelley>
 //                 Michal Adamczyk <https://github.com/mradamczyk>
 //                 Marvin Hagemeister <https://github.com/marvinhagemeister>
+//                 Josh McCullough <https://github.com/joshmccullough>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-type ClassValue = string | number | ClassDictionary | ClassArray | undefined | null | boolean;
```

Comment:
Could you just export the names are defined here instead of putting them in a new file?